internal/store/pod.go

```diff
@@ -57,8 +57,8 @@ var (
 
 				m := metric.Metric{
 
-					LabelKeys:   []string{"host_ip", "pod_ip", "uid", "node", "created_by_kind", "created_by_name", "priority_class"},
-					LabelValues: []string{p.Status.HostIP, p.Status.PodIP, string(p.UID), p.Spec.NodeName, createdByKind, createdByName, p.Spec.PriorityClassName},
+					LabelKeys:   []string{"host_ip", "pod_ip", "uid", "created_by_kind", "created_by_name", "priority_class"},
```

Comment:
This “info” type metric is ok to have it, just not all metrics. The idea is that any other metric can be easily joined with this one to add metadata when needed.

Reply:
@brancz thanks makes sense. So replacing here and removing from all other Pod metrics is sufficient?

Reply:
Yes I think so. I'll try and see if I can find any other cases in my review, but I believe this was all of them.